gateleen-core/src/main/java/org/swisspush/gateleen/core/storage/HttpResourceStorage.java

```diff
@@ -46,28 +48,29 @@ public void get(final String path, final Handler<Buffer> bodyHandler) {
         log.debug("Reading {}", path);
         client.request(HttpMethod.GET, path).onComplete(asyncResult -> {
             if (asyncResult.failed()) {
-                log.warn("Failed request to {}: {}", path, asyncResult.cause());
+                log.warn("Failed request to {}", path, new Exception("stacktrace", asyncResult.cause()));
+                log.error/*TODO*/("TODO why do we NOT report this error to bodyHandler?!? Is this"
+                        + " maybe the reason for all our mysterious timeouts everywhere? (ERR_jGQCALsBAgBJeA)");
```

Comment:
We should not add logs for personal bug hunting to the codebase. Just build your own release locally and add the logs there.